runelite-client/src/main/java/net/runelite/client/chat/ChatMessageManager.java

```diff
@@ -522,6 +538,16 @@ public void loadColors()
 			cacheColor(new ChatColor(ChatColorType.HIGHLIGHT, chatColorConfig.transparentFilteredHighlight(), true),
 				ChatMessageType.FILTERED);
 		}
+		if (chatColorConfig.transparentClanChatMessage() != null)
```

Comment:
I'm assuming these were meant for twitch chat config options that don't exist yet? ATM these two if blocks are duplicates.